src/config/configmanager.cpp

```diff
@@ -383,11 +383,17 @@ bool ConfigManager::reload() {
 	return result;
 }
 
+void ConfigManager::missingConfigWarning(const char* identifier) {
+	g_logger().warn("[ConfigManager::missingConfigWarning] Warning: Missing configuration for identifier: " + std::string(identifier));
```

Comment:
You could use fmt::format and __FUNCTION__ to get the name of the method instead of hardcode and omit the warning string in the message because the log will be a warning type, so it will be redundant.

```suggestion
	g_logger().warn(fmt::format("{}: Missing configuration for identifier: {}", __FUNCTION__, std::string(identifier)));
```